php/WP_CLI/Runner.php

```diff
@@ -73,7 +73,7 @@ class Runner {
 	private $required_files;
```

Comment:
Can we revert all of the formatting changes? It's hard to review the changes relevant to this PR...

Reply:
Sure.. sorry about that